client/src/app/pages/vulnerability-list/vulnerability-table.tsx

```diff
@@ -91,12 +91,12 @@ export const VulnerabilityTable: React.FC = () => {
         <Thead>
           <Tr>
             <TableHeaderContentWithControls {...tableControls}>
-              <Th {...getThProps({ columnKey: "identifier" })} />
-              <Th {...getThProps({ columnKey: "title" })} />
-              <Th {...getThProps({ columnKey: "severity" })} />
+              <Th {...getThProps({ columnKey: "name" })} />
```

Comment:
The `columnKey` should be filled with the name of the "property" that was defined in the `useTableControlState` hook.

When `useTableControlState` is used we write something like the image below:

![Screenshot from 2024-10-07 14-12-02](https://github.com/user-attachments/assets/95c8ee5d-e68c-4b68-9c61-d961e9b7999e)

The "columnNames" represent the columns the table will have. It as a [key, value] object. The KEY represents the name of the column, and the VALUE represents how that Column will be rendered.

Example:

```
 const tableControlState = useTableControlState({   
    columnNames: {
      name: "Nombre del cliente",      
    },
```


Then we should have something like:

```
<Th {...getThProps({ columnKey: "name" })} />
```

And the Browser will render "Nombre del cliente" in the table header